doc/manual/pragmas.txt

```diff
@@ -1087,3 +1087,68 @@ In the above example, providing the -d flag causes the symbol
 ``FooBar`` to be overwritten at compile time, printing out 42. If the
 ``-d:FooBar=42`` were to be omitted, the default value of 5 would be
 used.
+
+
+User custom pragmas 
+-------------------
+It is possible to define custom typed pragmas. Custom pragmas do not effect 
+code generation directly, but their presence can be detected by macros.
+Custom pragmas are defined using ``pragma`` keyword:
```

Comment:
Nitpick, but `pragma` isn't a keyword? Should be: "/.../ defined using the `pragma` pragma".

Great to see this feature implemented!